tooling/Common.csproj.props

```diff
@@ -0,0 +1,39 @@
+<Project>
+  <PropertyGroup>
+    <!-- Common C# Properties -->
+    <LangVersion>9.0</LangVersion>
+    <Features>strict</Features>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+
+    <!-- Common NuGet Package Properties -->
+    <Authors>Bonsai Foundation</Authors>
+    <Copyright>Copyright © Bonsai Foundation CIC and Contributors 2011-2024</Copyright>
+    <PackageProjectUrl>https://bonsai-rx.org</PackageProjectUrl>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
```

Comment:
Nice, didn't know about `PublishRepositoryUrl`. Just a nitpick for my OCD if we could move this line above so the URLs group together, or even better move the `PublishRepositoryUrl` to the bottom of this section.

Reply:
Yup, this is technically part of Source Link.

Haha yeah that was a little gross of me, it's at the end so it's not splitting the URLs anymore.